packages/salesforcedx-vscode-core/src/commands/functions/forceFunctionStart.ts

```diff
@@ -31,6 +31,37 @@ import {
 import { Uri, window } from 'vscode';
 import { FunctionService } from './functionService';
 
+/**
+ * Error types when running SFDX: Start Function
+ * This is also used as the telemetry log name.
+ */
+type ForceFunctionStartErrorType =
+  | 'force_function_start_plugin_not_installed'
+  | 'force_function_start_docker_plugin_not_installed_or_started';
+
+const forceFunctionStartErrorInfo: {
+  [key in ForceFunctionStartErrorType]: {
+    cliMessage: string;
+    cliExitCode: number;
+    errorNotificationMessage: string;
+  }
+} = {
+  force_function_start_plugin_not_installed: {
```

Comment:
Based on a slack discussion with @xyc , starting the same function multiple times kills the first one and starting a different function when another function has started throws a port-bind error. These are edge cases and we can take it up later.